pilot/pkg/config/kube/ingress/conversion.go

```diff
@@ -319,7 +329,8 @@ func shouldProcessIngressWithClass(mesh *meshconfig.MeshConfig, ingress *knetwor
 }
 
 func createFallbackStringMatch(s string) *networking.StringMatch {
-	if s == "" {
+	// If the string is empty or a wildcard, return nil
```

Comment:
I think in the current state `//*` was also broken. do you know if this resolve that as well?

Reply:
What was the error you were seeing when specifying //* as the path? Was it the same error? I think this path would have worked previously. The suffix /* would have been removed in `createFallbackStringMatch` and the remaining / would have been converted to a prefix match rather than a path-separated prefix match.

Reply:
oh sorry! it was "//.*"

Reply:
//.* is not addressed by the change. It would be converted to / which would fail the path separated prefix regex. It might be simplest to add a check to specifically handle this in route.go (similar to the single forward slash check), but I don't love this option. I'll try to come up with an alternative, but I don't want it to get too complicated.